### PR TITLE
Major and plans graphql

### PR DESCRIPTION
**Pull request by ghost**
Added schema and resolvers (rough) for Major requirements and plans of study, with Mitch.

Still needs:
* testing for resolvers
* imports for resolvers
* PR for database to be merged
* postgres to be added to prod

### CODE REVIEW

**Comment by ghost** [comment]
@gamburgm

**Comment by gamburgm** [comment]
Awesome!

Once the importing madness has been figured out with `Sequelize`, we'll throw up that PR, get it merged, and @dajinchu can set up Postgres, and this'll be next up to bat. Going to clean this up and add tests/whatnot to get the last pieces together. Thanks @bugsalexander !!!

File: backend/graphql/tests/major.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    requirements: { name: 'Biochemistry, BS', yearVersion: 2018 },
+    plansOfStudy: [ { years: [ 1000 ], id: '0' } ],
+  });
+});
+
+afterAll(async () => {
+  await db.sequelize.close();
+});
+
+it('gets major from majorId', async () => {
```
Comment (dajinchu) [inline backend/graphql/tests/major.test.js:42]: Add a test for when the majorId is invalid

File: backend/graphql/tests/major.test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  expect(res).toMatchSnapshot();
+});
+
+it('gets specific occurrence', async () => {
+  const res = await query({
+    query: gql`
+      query major {
+        major(majorId: "computer-information-science/computer-science/bscs") {
+          name
+          occurrence(year: 2017) {
```
Comment (dajinchu) [inline backend/graphql/tests/major.test.js:61]: add test for when year is invalid

**Review by dajinchu — CHANGES_REQUESTED** [review]
looks good, just needs a couple tests

**Review by dajinchu — APPROVED** [review]
sweeeet

**Pull request merged**